### PR TITLE
BET-969 Add stop/exception attrs and test metadata

### PR DESCRIPTION
**Pull request by rodrigues**
https://bet-project.atlassian.net/browse/BET-969

Follow up from https://github.com/bet01/bettelemetry/pull/12

### CODE REVIEW

File: test/instrumentation_test.exs [hunk 1]
```diff
@@ -144,8 +213,62 @@ defmodule BetTelemetry.InstrumentationTest do
...
 
-      assert_receive {:span,
-                      span(name: "my_namespace.my_event", attributes: [{:success, false}, {_, _}])}
+      assert_receive {:span, span(name: "my_namespace.my_event", attributes: attrs)}
+
+      refute attrs[:success]
+      assert attrs[:kind] == :error
+      assert attrs[:reason] == %RuntimeError{message: "Crash!"}
+
+      assert [{__MODULE__, _, 1, file: 'test/instrumentation_test.exs', line: 211} | _] =
```
Comment (lsigauke) [inline test/instrumentation_test.exs:222]: I think maybe we should ignore the line number? This test will fail if we add anything into this file

**Review by lsigauke — COMMENTED** [review]
Looks good, just left one comment there

File: test/instrumentation_test.exs [hunk 2]
```diff
@@ -9,6 +9,22 @@ defmodule BetTelemetry.InstrumentationTest do
...
+    d: {:error, :oops},
+    e: [1, 2, 3],
+    f: %{chatta: "nooga"},
+    g: [%{a: 1}],
+    h: DateTime.utc_now()
+  }
+
+  def attr_value(attrs, key) do
+    Enum.find_value(attrs, fn {k, v} -> k == key && v end)
+  end
```
Comment (gilacost) [inline test/instrumentation_test.exs:26]: This is not obvious at all, would be better to have a signature per case 😅

Comment (rodrigues) [reply test/instrumentation_test.exs:26]: not sure what you're suggesting

Comment (gilacost) [reply test/instrumentation_test.exs:26]: Sorry for confusing you, I might not understand what you are trying to do here, what is this function trying to do 🤔 ?

**Pull request merged**